File: pkg/models/trait.go
package models

type Trait struct {
	Name        string `json:"name,omitempty"`
	Slug        string `json:"slug,omitempty"`
	ID          int    `json:"id,omitempty"`
	Description string `json:"description,omitempty"`
	//IsLargeTrait   bool `json:"isLargeTrait,omitempty"`
	//RequiredPoints int    `json:"requiredPoints,omitempty"`
	//RequiredLevel  int    `json:"requiredLevel,omitempty"`
}

File: client/src/interfaces/trait.ts
export interface TraitData {
    name: string
    slug: string
    id: number
    description: string
}

export interface TraitOfMonsterData {
    name: string
    slug: string
    id: number
    description: string
    isLargeTrait: boolean
    requiredLevel: number
}

export interface TraitOfTalentData {
    name: string
    slug: string
    id: number
    description: string
    requiredPoints: number
}